angular-workspace/projects/ni/nimble-angular/src/directives/list-option/nimble-combobox-list-option.directive.ts

```diff
@@ -26,7 +26,8 @@ export class NimbleComboboxListOptionDirective implements AfterViewInit, OnDestr
     public set ngValue(value: unknown) {
         if (this.combobox) {
             this._modelValue = value;
-            this.updateComboboxValue(value);
+            this.changeDetector.detectChanges();
```

Comment:
Guessing this will change a bit after https://github.com/ni/nimble/pull/1470
So can take a look again after that pone merges

Reply:
Updated.